snmp/datadog_checks/snmp/data/default_profiles/dell-poweredge.yaml

```
@@ -28,6 +28,8 @@ metrics:
       OID: 1.3.6.1.4.1.674.10892.1.200.10
       name: systemStateTable
     symbols:
+      - name: systemStatePowerSupply
```

```suggestion
      - name: systemState
```

Same question as https://github.com/DataDog/integrations-core/pull/15456/files#r1283477336